vulnerabilities/importers/github_osv.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+import json
+import logging
+from io import BytesIO
+from pathlib import Path
+from typing import Iterable
+from zipfile import ZipFile
+
+import requests
+
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import Importer
+from vulnerabilities.importers.osv import parse_advisory_data
+
+logger = logging.getLogger(__name__)
+
+
+class GithubOSVImporter(Importer):
+    license_url = "https://github.com/github/advisory-database/blob/main/LICENSE.md"
+    spdx_license_expression = "CC-BY-4.0"
+    url = "https://codeload.github.com/github/advisory-database/zip/refs/heads/main"
+
+    def advisory_data(self) -> Iterable[AdvisoryData]:
+        response = requests.get(self.url).content
+        with ZipFile(BytesIO(response)) as zip_file:
+            for file_name in filter(
+                lambda filename_list: ("github-reviewed" in Path(filename_list).parts)
+                and Path(filename_list).suffix == ".json",
+                zip_file.namelist(),
+            ):
+                with zip_file.open(file_name) as f:
+                    raw_data = json.load(f)
+                    try:
+                        yield parse_advisory_data(
+                            raw_data,
+                            supported_ecosystems=[
```

Comment:
support rubygems as well

Reply:
I think I should add mapper for ecosystems , OSV relay on `RANGE_CLASS_BY_SCHEMES`  : `gems` but osv call the  ecosystem : `rubygems`